model_cards/Naveen-k/KanBERTo/README.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to KanBERTo (ಕನ್ಬರ್ಟೋ)
+
+## Model Description
+ 
+> This is a small language model for [Kannada](https://en.wikipedia.org/wiki/Kannada) language with 1M data samples taken from
+  [OSCAR page](https://traces1.inria.fr/oscar/files/compressed-orig/kn.txt.gz)
+
+## Training params 
+
+- **Dataset** - 1M data samples are used to train this model from OSCAR page(https://traces1.inria.fr/oscar/) eventhough data set is of 1.7 GB due to resource constraint to train 
+I have picked only 1M data from the total 1.7GB data set. If you are interested in collaboration and have computational resources to train on you are most welcome to do so.
```

Comment:
have you tried running on Google Colab? and/or Colab Pro?

Reply:
Hi @julien-c , Yes i have tried training with the same data set with smaller size and was able to train model from it.